### PR TITLE
adding procedure to create mirror for RHEL without Satellite server

### PR DESCRIPTION
**Pull request by emuus**

### CODE REVIEW

File: admin/en/source/pages/upgrade/upgrade-overview.rst [hunk 1]
```diff
@@ -123,6 +123,65 @@ Run the following CLI command in order to know if Squid is running:
 
 		$ service squid start	
 
+.. _redhat-mirror:
+
+Creating a Local Mirror for Red Hat Without Using Satellite Server
```
Comment (ussmarc) [inline admin/en/source/pages/upgrade/upgrade-overview.rst]: Creating a local mirror for Red Hat without using a Satellite server

File: admin/en/source/pages/upgrade/upgrade-overview.rst [hunk 2]
```diff
@@ -123,6 +123,65 @@ Run the following CLI command in order to know if Squid is running:
...
+------------------------------------------------------------------
+
+You can create a local mirror of the latest updates for Red Hat Enterprise Linux without using satellite server. How this is set up will depend on your version of Red Hat Enterprise Linux.
```
Comment (ussmarc) [inline admin/en/source/pages/upgrade/upgrade-overview.rst]: You can create a local mirror of the latest updates for Red Hat Enterprise Linux without using Satellite server. How this is set up will depend on your version of Red Hat Enterprise Linux.

**Comment by emuus** [comment]
This fixes issue UFORGE-7382

File: admin/en/source/pages/upgrade/upgrade-overview.rst [hunk 3]
```diff
@@ -123,6 +123,65 @@ Run the following CLI command in order to know if Squid is running:
...
+	* `How to register and subscribe a system to the Red Hat Customer Portal using Red Hat Subscription-Manager <https://access.redhat.com/solutions/253273>`_.
+
+Red Hat 5 and 6
+~~~~~~~~~~~~~~~
+
+1. Install Red Hat Enterprise Linux 5.0 or 6.0.
+
+2. Register the RH system running and register the system::
+
+	rhn_register command
```
Comment (ussmarc) [inline admin/en/source/pages/upgrade/upgrade-overview.rst]: `rhn_register`

**Pull request merged**